build.sbt

```diff
@@ -3,12 +3,12 @@ import de.johoop.jacoco4sbt.XMLReport
 val driverVersion = "1.1.1-SNAPSHOT"
 val asyncHttpClientVersion = "1.9.39"
 val guavaVersion = "19.0"
-val jacksonVersion = "2.6.4"
-val jacksonDocVersion = "2.6"
+val jacksonVersion = "2.8.8"
```

Comment:
I looked through the jackson changelog and it seems fine. But someone else should look and confirm that there isn't an unintended consequence of this upgrade.

I'm curious: are we upgrading jackson here for scala version compatability reasons?